src/screens/AddNewItemScreen/hooks/usePress.ts

```diff
@@ -0,0 +1,67 @@
+import { stopNotificationHelper } from '../../../helpers/stopNotificationHelper';
+import { createNewItemHelper } from '../../../helpers/createNewItemHelper';
+import { createNotificationHelper } from '../../../helpers/createNotificationHelper';
+import { editItemSagaAction } from '../../../redux/actions/todoSagaActions/editItemSagaAction';
+import { useDispatch, useSelector } from 'react-redux';
+import { TodoItemType } from '../../../models';
+import { deviceTokenSelector } from '../../../redux/selectors/deviceTokenSelector';
+import { useState } from 'react';
+import {
+  addItemSagaAction,
+  AddItemSagaActionPayload,
+} from '../../../redux/actions/todoSagaActions/addItemSagaAction';
+import { useNavigation } from '@react-navigation/native';
+import { AddNewItemScreenNavigationProps } from '../type';
+import { userTokenSelector } from '../../../redux/selectors/userTokenSelector';
+
+export type UsePressParams = {
+  isEdit: boolean;
+  editItem: TodoItemType;
+  editText: TodoItemType['text'];
+  editDate: Date;
+};
+export const usePress = (params: UsePressParams) => {
```

Comment:
create type for whole hook (with params and return array)

Reply:
ok